src/layers/conv.jl

```diff
@@ -379,8 +422,24 @@ end
 """
     CrossCor(weight::AbstractArray, [bias, activation; stride, pad, dilation])
 
-Constructs a layer with the given weight and bias arrays.
-Accepts the same keywords as the `CrossCor((4,4), 3 => 7, relu)` method.
+Constructs a CrossCor layer with the given weight and bias.
+Accepts the same keywords (and has the same defaults) as the `CrossCor((5,5), 3 => 6, relu)` method.
```

Comment:
```suggestion
Accepts the same keywords and has the same defaults as `CrossCor((5,5), 3 => 6, relu)`.
```
I would trim these bits off the other Conv* constructor doc strings as well.